include/tvm/runtime/vm.h

```diff
@@ -286,6 +298,15 @@ struct Instruction {
    */
   static Instruction InvokePacked(Index packed_index, Index arity, Index output_size,
                                   const std::vector<RegName>& args);
+  /*! \brief Construct an invoke external instruction.
+   *  \param packed_index The index of the external function.
+   *  \param ext_arity The arity of the function.
+   *  \param ext_output_size The number of outputs of the external function.
+   *  \param args The argument registers.
+   *  \return The invoke external instruction.
+   */
+  static Instruction InvokeExternal(Index external_index, Index ext_arity, Index ext_output_size,
```

Comment:
Do we actually need a second instruction, shouldn't calling externals be opaque to the VM?